common/supplemental/likelySubtags.xml

```diff
@@ -2882,6 +2897,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Zaza; ?; ? } => { Zaza; Latin; Türkiye }-->
 		<likelySubtag from="und" to="en_Latn_US"/>
 		<!--{ ?; ?; ? } => { English; Latin; United States }-->
+		<likelySubtag from="und_001" to="en_Latn_US"/>
```

Comment:
Hmm, I do not offhand know of another case in which likelySubtags alters an explicit subtag other than und or ZZ. This seems a bit odd. Why not und_001 -> en_Latn_001 (since we have a locale for en_001)?

Reply:
There are other nnn to TT (number to territory ) cases. But yeah en 101 makes sense here

Reply:
The other ones are continents; 001 is the whole world.